gym/spaces/graph.py

```diff
@@ -0,0 +1,260 @@
+"""Implementation of a space that represents graph information where nodes and edges can be represented with euclidean space."""
+from typing import Optional, Sequence, Union
+
+import numpy as np
+
+from gym.spaces.box import Box
+from gym.spaces.discrete import Discrete
+from gym.spaces.multi_discrete import MultiDiscrete
+from gym.spaces.space import Space
+from gym.utils import seeding
+
+
+class GraphObj:
+    r"""A base for constructing information as graphs."""
+
+    def __init__(
+        self,
+        nodes: np.ndarray,
+        edges: Optional[np.ndarray] = None,
+        edge_links: Optional[np.ndarray] = None,
+    ):
+        r"""Constructor for Graph information.
```

Comment:
Could you use the google docstring style?

Reply:
Sure